ts/WoltLabSuite/Core/Element/woltlab-core-dialog-control.ts

```diff
@@ -144,19 +144,16 @@ export class WoltlabCoreDialogControlElement extends HTMLElement {
       this.append(this.#extraButton);
     }
   }
+}
 
-  public addEventListener<T extends keyof WoltlabCoreDialogControlEventMap>(
-    type: T,
-    listener: (this: WoltlabCoreDialogControlElement, ev: WoltlabCoreDialogControlEventMap[T]) => any,
-    options?: boolean | AddEventListenerOptions,
-  ): void;
-  public addEventListener(
-    type: string,
-    listener: (this: WoltlabCoreDialogControlElement, ev: Event) => any,
-    options?: boolean | AddEventListenerOptions,
-  ): void {
```

Comment:
As additional context: Keep in mind that for overloaded functions, the actual signature is not visible to the outside, unless it is repeated.